Products/zms/conf/metaobj_manager/com.zms.foundation/ZMSTable/interface0.zpt

```diff
@@ -575,7 +581,7 @@
 						// Remove table with inline-links (if exists).
 						$('table',this).remove();
 						$('textarea,input',this).each(function() {
-								var key = this.id.substring(0,this.id.indexOf('_'));
+								var key = this.id.substr(0,this.id.indexOf('_'));
```

Comment:
substr() is Deprecated: This feature is no longer recommended.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr?retiredLocale=de